leash.go

```diff
@@ -516,3 +519,72 @@ func logStats(stats *responseStats, interval uint) {
 		stats.logAndReset()
 	}
 }
+
+// Final helper for outputting helpful log lines at the end of honeytail
+// execution. Calls private `responseStats` members indiscriminately
+// because we know this is only called at the end, after all goroutines
+// have exited.
+func outputHelp(options GlobalOptions, stats *responseStats, teamSlug string) {
+	// Build up Honeycomb URL for nicer logs
+	u, _ := url.Parse("https://ui.honeycomb.io")
```

Comment:
_wishes we had a :badpokerface: emoji in github_

I really wanted to be clever and `s/api/ui/` in the `options.Reqs.APIHost` value but it frankly seemed unnecessary since the **intent** is for external folks to use it =\

... Or maybe we peek at `APIHost` and if it's not `api.honeycomb.io`, then we just bail on trying to provide direct URLs in the first place.

Reply:
> wishes we had a :badpokerface: emoji in github

:)

Yeah, in these cases I think it's OK to just hardcode `ui.honeycomb.io` -- anyone sending to a different API host should probably know how to get to the dataset for that clandestine API on their own, i.e., it's most likely Honeycomb employees+contractors.